forge/routes/api/assistant.js

```diff
@@ -63,6 +63,22 @@ module.exports = async function (app) {
 
         // post to the assistant service
         const headers = {}
+        if (request.session?.ownerType) {
+            switch (request.session.ownerType) {
+            case 'team':
+                headers['ff-owner-type'] = 'team'
+                headers['ff-owner-id'] = app.db.models.Team.encodeHashid(request.session.ownerId)
+                break
+            case 'device':
+                headers['ff-owner-type'] = 'device'
+                headers['ff-owner-id'] = app.db.models.Device.encodeHashid(request.session.ownerId)
+                break
+            case 'project':
+            case 'instance':
+                headers['ff-owner-type'] = request.session.ownerType
+                headers['ff-owner-id'] = request.session.ownerId
```

Comment:
```suggestion
            case 'instance':
                headers['ff-owner-type'] = request.session.ownerType
                headers['ff-owner-id'] = request.session.ownerId
                break
            default:
                // Don't log the id as we don't know how to hash it
                // Log the type so we can spot cases we aren't handling and address it
                response.request.ownerType = reply.request?.session?.ownerType
```

Apply same default as logging

Reply:
discussed. not required.